config/initializers/devise.rb

```diff
@@ -24,10 +24,11 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = ENV["CORE_EMAIL_USERNAME"]
+  # config.mailer_sender = ENV["CORE_EMAIL_USERNAME"]
```

Comment:
This could just be deleted right? Since this was only set up for the Gmail

Reply:
Possibly. Might still be useful as a fallback for local dev or other things? At the moment only devise emails are routed through Notify.